myia/compile/debug_lin.py

```diff
@@ -0,0 +1,70 @@
+"""Linear implementation using the debug VM."""
+
+from .utils import get_outputs
+
+from ..ir import Graph, is_apply, is_constant, manage
+from ..prim import Primitive, vm_implementations
+from ..prim.ops import cons_tuple
+from ..vm import VM
+
+
+def make_tuple(vals, g):
+    """Make a graph that builds a tuple from the specified values."""
+    tup = g.constant(())
+    for v in reversed(list(vals)):
+        tup = g.apply(cons_tuple, v, tup)
+    return tup
```

Comment:
I had to implement the same helper in the grad branch, so this could be useful for other code. Could you move it to ir.utils?

Reply:
I agree it should be shared, but since it depends on prim.ops.cons_tuple, I'm not sure it should be under ir/.

Maybe graph_utils?